types.js

```diff
@@ -2,7 +2,8 @@ module.exports = {
   CHOOSE_ONE: 'chooseOne',
   ERROR_POSITION_TYPE: 'ERROR_POSITION_TYPE',
   ERROR_POSITION_LATE: 'ERROR_POSITION_LATE',
-  ERROR_POSITION_CHOICE: 'ERROR_POSITION_CHOICE'
+  ERROR_POSITION_CHOICE: 'ERROR_POSITION_CHOICE',
+  SCHEMA_VERSION: 'v1'
```

Comment:
caution here ... it could be easy to get out of sync with ssb-poll-schema.
I think I want something either programmatic here which pulls the version from the ssb-poll-schema, or a big note saying "manage this carefully, make sure you're in sync with the poll-schema spec you're aiming for"

Reply:
Yeah. 
I think this would feel good if `ssb-poll-schema` exported a constant `SSB_POLL_SCHEMA_V1_STRING` etc.

Reply:
done. added as part of this pr